browser/css/btns.css

```
@@ -1,4 +1,4 @@
-.jsdialog-container .ui-button-box button,
+ffont.jsdialog-container .ui-button-box button,
```

```suggestion
.jsdialog-container .ui-button-box button,
```

That is a typo I guess?

---

thanks a lot